containers/fedora/build.sh

```
@@ -71,22 +71,59 @@ if [ $FEDORA_VERSION -gt 40 ]; then
+        virsh undefine --nvram "${NAME}"
+    else
+        virsh undefine "${NAME}"
+    fi
+fi
+
 echo "Run the VM (ctrl+] to exit)"
 virt-install \
   --memory 2048 \
   --vcpus 2 \
   --arch $CPU_ARCH_CODE \
   --name $NAME \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)